mllib/src/test/scala/org/apache/spark/ml/evaluation/RegressionEvaluatorSuite.scala

```diff
@@ -65,15 +65,15 @@ class RegressionEvaluatorSuite
 
     // default = rmse
     val evaluator = new RegressionEvaluator()
-    assert(evaluator.evaluate(predictions) ~== 0.1019382 absTol 0.001)
+    assert(evaluator.evaluate(predictions) ~== 0.1013829 absTol 0.001)
```

Comment:
Your change implies the thresholds are a bit wrong though (or else your change is wrong, but I don't think it is). I don't see how that's a separate issue, no. That is, the thresholds + expected result should also have passed the original implementation, if we believe it was correct too.